src/components/Tooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+interface TooltipProps {
```

Comment:
Also can I know why we are abstracting just this tooltips component into a separate file, I understand it helps readability but apart from that is there any specific requirement in doing this @Kishan1835  ?